src/ext/oblivc/obliv_bits.c

```diff
@@ -56,7 +56,6 @@ static struct stdioTransport stdioTransport
 void protocolUseStdio(ProtocolDesc* pd)
   { pd->trans = &stdioTransport.cb; }
 
-//#define PROFILE_NETWORK
```

Comment:
Runtime flag is (as I feared) actually much worse than simply doing the additions. If you approve it as the right solution, I'll revert the network profiling changes and add another commit to the PR.